packages/mdctl-core/streams/section.js

```diff
@@ -37,27 +37,32 @@ const _ = require('lodash'),
         },
       },
       NON_WRITABLE_KEYS = ['facet'],
-      SectionsCreated = [],
       { privatesAccessor } = require('@medable/mdctl-core-utils/privates'),
       crypto = require('crypto')
 
 class ExportSection {
 
-  constructor(content, key = '') {
+  constructor(content, key = '', sectionsWithResources) {
```

Comment:
add sectionsWithResources = [] as default value otherwise it could fail after in the extractAssets.